server.py

```diff
@@ -118,24 +177,48 @@ def get_verified_email(token):
     except Exception:
         return {'error': 'Invalid token'}
 
+    # Take the email address from the 'subject' claim in the payload, and
+    # verify the format of the email address is what we expect.
     sub = payload['sub']
     if not re.match('.+@.+', sub):  # <-- TODO: Use a proper parser.
```

Comment:
`emailaddress` does exactly as much is this right now, so will just get rid of the TODO.